core/src/main/res/layout/mz_view_map.xml

```diff
@@ -11,6 +11,20 @@
       android:layout_height="match_parent"
       />
 
+  <com.mapzen.android.graphics.CompassView
+      android:id="@+id/mz_compass"
+      android:layout_width="@dimen/mz_compass_button_width"
+      android:layout_height="@dimen/mz_compass_button_height"
+      android:layout_alignParentLeft="true"
```

Comment:
+1 for providing both overlay configuration options.

I don't mind merging this implementation as-is then re-introducing the "classic" overlay layout option for Eraser Map and other apps that would prefer the original button placement.